_locales/en/messages.json

```
@@ -242,15 +242,15 @@
   "extensionHasUpdated": {
     "message": "Scratch Addons has updated to v$1"
   },
-  "extensionUpdateInfo1_v1_37": {
-    "message": "New addon: \"project page download button\", which allows you to download projects directly from the project page, without having to open the editor first. Go to $1 to enable it.",
-    "description": "The placeholder is \"Scratch Addons settings\"."
+  "extensionUpdateInfo1_v1_38": {
+    "message": "We have made changes behind-the-scenes to support the new extension architecture required by browsers (Manifest V3). If you encounter any issues, please $1. There is also a button to send feedback in the settings page.",
```

My proposed update description taking the gradual rollout into account:
```suggestion
    "message": "This is an experimental update that we're releasing to a portion of our user base. It contains some behind-the-scenes changes to support the new extension architecture required by browsers (Manifest V3). If you encounter any issues or unexpected behavior, please $1. There is also a button to send feedback in the settings page.",
```

---

But what if we don't ship a .1 release and all users end up getting this "experimental" update message?

---

> But what if we don't end up shipping a patch release and all users end up getting this "experimental" update message?

Then we could change it.

---

> But what if we don't end up shipping a patch release and all users end up getting this "experimental" update message?

It's sometimes fine to lie

---

I think I prefer the current one then.